UnityProject/Assets/Scripts/UI/UI Bottom/UI_ItemSwap.cs

```diff
@@ -34,7 +34,8 @@ public void OnClick()
 			return;
 		}
 
-		SoundManager.Play("Click01");
+		// JESTER
+		//SoundManager.Play("Click01");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)